tests/unit/setup-rendering-context-test.js

```diff
@@ -28,7 +28,7 @@ module('setupRenderingContext', function (hooks) {
     return;
   }
 
-  hooks.after(function () {
+  hooks.afterEach(function () {
```

Comment:
Using `hooks.after` here meant that the tests _within_ this module were not able to assume `owner.visit` was available (because when `setApplication(null)` is ran, we use a "fake owner" which doesn't have the ability to `visit`).